server/src/test/java/umm3601/todo/TodoControllerSpec.java

```diff
@@ -0,0 +1,144 @@
+package umm3601.todo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+import io.javalin.core.validation.Validator;
+import io.javalin.http.BadRequestResponse;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+import umm3601.Server;
+
+/**
+ * Tests the logic of the TodoController
+ *
+ * @throws IOException
+ */
+public class TodoControllerSpec {
+
+  private Context ctx = mock(Context.class);
+
+  private TodoController TodoController;
+  private static TodoDatabase db;
+
+  @BeforeEach
+  public void setUp() throws IOException {
+    ctx.clearCookieStore();
+
+    db = new TodoDatabase(Server.TODO_DATA_FILE);
+    TodoController = new TodoController(db);
```

Comment:
Is it the convention to use a capitalized name for an instance of a controller? I would have guessed not, but would not be surprised if I'm wrong.